raml-sanitize.js

```diff
@@ -172,6 +172,11 @@ function toSanitization (configs, rules, types) {
         value = value[0]
       }
 
+      // Support RAML 1.0 array types for single values.
+      if (Array.isArray(config.type) && config.type.length === 1 && config.type[0] === 'array' && !Array.isArray(value)) {
```

Comment:
Please assign these assertions to variables with human-understandable names and use those in the `if` statement.
E.g. 
```js
var aliensArrived = aliens.army.length > 3
var theyAreMartians = aliens.race === 'martian'
if (aliensArrived && theyAreMartians) {
    console.log('Martians arrived')
}
```
Because it's difficult to understand right away what assertions like `config.type.length === 1` mean.
Also this will help keep a proper line length (at about < 80).